### ISSUE TITLE
make start fails to start webpack-dev-server for local development.

### ISSUE THREAD

**Issue by iam-veeramalla** [argoproj/argo-cd#4968]
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**
`make start` fails to start webpack-dev-server for local development. I am using mac os for my local development.

**To Reproduce**
`make start` for the very first time to bring up the webpack-dev-server. 

**Expected behavior**
I should be able to bring up UI.

**Work Around**
Run `make dep-ui` and then run `make start`

**Version**

```shell
argocd: v1.7.10+bcb05b0.dirty
  BuildDate: 2020-11-20T23:40:45Z
  GitCommit: bcb05b0c2e0f8006aa2d2abaf780e73c9e73c945
  GitTreeState: dirty
  GoVersion: go1.15.5
  Compiler: gc
  Platform: darwin/amd64
```

**Logs**

```
12:08:13          ui | yarn run v1.22.4
12:08:15          ui | $ webpack-dev-server --config ./src/app/webpack.config.js --mode development
12:08:16          ui | /bin/sh: 1: webpack-dev-server: not found
```

**Comment by iam-veeramalla** [comment]
Work Around
Run `make dep-ui` and then run `make start`

**Comment by jessesuen** [comment]
@iam-veeramalla would you like to update Makefile to do this?

**Comment by iam-veeramalla** [comment]
> @iam-veeramalla would you like to update Makefile to do this?

Sure .. I will raise a PR for this .. PR On the way 👍

**Comment by jannfis** [comment]
Closed via #4990
